Microsoft.Toolkit.Uwp.SampleApp/SamplePages/WrapLayout/WrapLayoutPage.xaml.cs

```diff
@@ -40,6 +45,10 @@ public void OnXamlRendered(FrameworkElement control)
             {
                 repeater.ItemsSource = _items;
             }
+
+            _wrapScrollParent = control.FindChildByName("WrapScrollParent") as ScrollViewer;
+            var wrapLayoutParent = _wrapScrollParent?.FindDescendant<ItemsRepeater>();
+            _wrapLayout = wrapLayoutParent.Layout as WrapLayout;
```

Comment:
I hit an exception here, we should already have a reference to the repeater above, testing something out quick.